debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/MethodMetaData.java

```diff
@@ -451,43 +448,23 @@ public void jitAddSpilledRegisters(WalkState walkState, VoidPointer stackMap) th
 							walkState.registerEAs[mapCursor] = saveCursor;
 							saveCursor = saveCursor.add(1);
 						}
-						else if (md.registerSaveDescription().bitAnd(0xFFFF0000L).eq(new UDATA(J9TR_SHRINK_WRAP).bitAnd(new UDATA(0xFFFFFFFFL)))) /* mask for 32-bit compare */
-						{
-							/* preserved registers: 
-							 * rbx [2], r9-r15 [10-16] on 64-bit
-							 * ebx [2], ecx [3], esi [6] on 32-bit
-							 */
-
-							/* make sure the stack slots are checked in lockstep
-							 * and bump the saveCursor only if the register is marked 
-							 */
-							if (registersShrinkWrapped.anyBitsIn(1)) 
-								saveCursor = saveCursor.add(1);
-						}
 						i = i.add(1);
 						++mapCursor;
 						registerSaveDescription = registerSaveDescription.rightShift(1);
-						registersShrinkWrapped = registersShrinkWrapped.rightShift(1);
 					}
 					while (! registerSaveDescription.eq(0));
 				}
 			} else if (TRBuildFlags.host_POWER || TRBuildFlags.host_MIPS) {
 				if (TRBuildFlags.host_POWER) {
-					if (true) { /*defined(TR_SHRINK_WRAP)*/
-						/*
-						 * see PPCLinkage for a description of the RSD 
-						 * the save offset is located from bits 18-32 
-						 * so first mask it off to get the bit vector 
-						 * corresponding to the saved GPRS
-						 */
-						savedGPRs = registerSaveDescription.bitAnd(new UDATA(0x1FFFFL));
-						saveOffset = registerSaveDescription.rightShift(17).bitAnd(new UDATA(0xFFFFL));
-						lowestRegister = new UDATA(15); /* gpr15 is the first saved GPR, so move 15 spaces */
-					} else {
-						savedGPRs = registerSaveDescription.bitAnd(new UDATA(31));
-						saveOffset = registerSaveDescription.rightShift(11);
-						lowestRegister = new UDATA(32).sub(savedGPRs);
-					}
+					/*
+						* see PPCLinkage for a description of the RSD 
+						* the save offset is located from bits 18-32 
+						* so first mask it off to get the bit vector 
+						* corresponding to the saved GPRS
+						*/
```

Comment:
Could you please fix the indentation of this comment so is consistent?

Reply:
Fixed in 19913fe